src/firestore/firestore.ts

```diff
@@ -111,5 +111,8 @@ function initFirestore(app: FirebaseApp): Firestore {
         'to use Cloud Firestore API.',
     });
   }
-  return new Firestore(options);
+
+  // Lazy-load the Firestore implementation here, which in turns loads gRPC.
+  let firestoreDatabase = require('@google-cloud/firestore');
```

Comment:
So, if I follow this correctly, line 21 merely loads the types?

Reply:
Yes. TS compiler won't emit a require when the imported thing is only used as a type annotation.